roles/contiv_network/tasks/ovs.yml

```diff
@@ -38,38 +38,16 @@
 
 # Remote install
 
-- name: get openstack ocata repo
+- name: get the openvswitch RPM
   get_url:
-    url: https://repos.fedorapeople.org/repos/openstack/openstack-ocata/rdo-release-ocata-2.noarch.rpm
-    dest: /tmp/rdo-release-ocata-2.noarch.rpm
-    validate_certs: false
+    url: https://cbs.centos.org/kojifiles/packages/openvswitch/2.5.0/2.el7/x86_64/openvswitch-2.5.0-2.el7.x86_64.rpm
```

Comment:
There is no guarantee that this repo/personal folder won't disappear tomorrow.

We need to host the file in our own repo. We might already have this as part of openstack repo that Metacloud folks host. Let's use that repo to fix this issue the right way.

Reply:
Currently we don't have a produce repo for public access. I have ticket created to track this work. In the meantime, @rhim suggests to put OVS 2.5 rpm in github to mitigate the issue. 

For the longer term, we need to setup our public accessible repo to lock down the dependency versions.

Reply:
I've updated the PR to have two locations. It'll try to fetch the RPM from this location first and fall back to GitHub. This should be fine. I've also included a file hash to make sure it's the same file.